docs/api/enum/WdTabAlignment.rst

```diff
@@ -0,0 +1,38 @@
+.. _WdTabAlignment:
+
+``WD_TAB_ALIGNMENT``
+====================
+
+Specifies the alignment of tab stops that can be applied to a paragraph.
+
+----
+
+LEFT
+    Left-aligned.
+
+CENTER
+    Center-aligned.
+
+RIGHT
+    Right-aligned.
+
+DECIMAL
+    Decimal-aligned.
+
+BAR
+    Bar-aligned.
+
+LIST
+    List-aligned. (deprecated)
+
+START
+    Left-aligned.  (deprecated)
+
+END
+    Right-aligned.  (deprecated)
+
+NUM
+    Left-aligned!  (deprecated)
+
+CLEAR
+    Clear an inherited tab stop.
```

Comment:
I don't see these enumerations being referenced from the tests, so they shouldn't be part of the commit. In general, these are added on the first commit where they raise an error if they're not there.